src/instructlab/lab.py

```diff
@@ -712,15 +712,37 @@ def chat(
 def download(ctx, repository, release, filename, model_dir):
     """Download the model(s) to train"""
     click.echo(f"Downloading model from {repository}@{release} to {model_dir}...")
+    if (
+        "HF_TOKEN" not in os.environ
+        and repository != "instructlab/merlinite-7b-lab-GGUF"
+    ):
+        raise ValueError(
+            "HF_TOKEN var needs to be set in your environment to download HF Model"
```

Comment:
Should we also update the readme user docs which point to the additional steps (this and stuff like model-family etc)? They can be a separate PR in the interest of time. For now maybe we can make the error message more descriptive

```suggestion
            "HF_TOKEN var needs to be set in your environment to download HF Model. The HF Token is used to authenticate your identity to the Hugginface Hub."
```

Reply:
small nit in case you commit this: 
's/Hugginface/Huggingface/'